packages/shared/src/components/squads/PublicSquadEligibilityCard.tsx

```diff
@@ -1,59 +1,65 @@
+import React, { ReactElement, useCallback } from 'react';
+import { useRouter } from 'next/router';
+import {
+  useActions,
+  useFeedLayout,
+  useSquad,
+  usePublicSquadRequests,
+} from '../../hooks';
 import { Button } from '../buttons/Button';
 import { ButtonSize, ButtonVariant } from '../buttons/common';
 import { Card } from '../cards/Card';
 import { Card as CardV1 } from '../cards/v1/Card';
 import { EarthIcon, MiniCloseIcon, TimerIcon } from '../icons';
-import { ProgressBar } from '../fields/ProgressBar';
 import { ActionType } from '../../graphql/actions';
 import { useLazyModal } from '../../hooks/useLazyModal';
 import { LazyModal } from '../modals/common/types';
+import { anchorDefaultRel } from '../../lib/strings';
+import {
+  MIN_SQUAD_POSTS,
+  SquadPostsProgressBar,
+} from './SquadPostsProgressBar';
+import { squadsPublicGuide } from '../../lib/constants';
+import { SquadStatus } from './settings';
+import { PlaceholderCard } from '../cards/PlaceholderCard';
 
-const MIN_POSTS = 3;
-
-interface Props {
-  postsCount: number;
-  inReview: boolean;
-  squadId: string;
-}
-
-const PublicSquadEligibilityCard = ({
-  postsCount,
-  inReview,
-  squadId,
-}: Props): ReactElement => {
+export function PublicSquadEligibilityCard(): ReactElement {
+  const router = useRouter();
+  const { squad } = useSquad({ handle: router.query.handle as string });
+  const { isFetched, status } = usePublicSquadRequests({
+    sourceId: squad?.id,
+  });
+  const postsCount = squad?.flags?.totalPosts;
   const { shouldUseListFeedLayout } = useFeedLayout();
-  const { completeAction, checkHasCompleted, isActionsFetched } = useActions();
+  const { completeAction } = useActions();
   const { openModal } = useLazyModal();
-  const hideCard = useMemo(
-    () =>
-      isActionsFetched &&
-      checkHasCompleted(ActionType.HidePublicSquadEligibilityCard),
-    [checkHasCompleted, isActionsFetched],
-  );
 
   const onDismiss = useCallback(() => {
     completeAction(ActionType.HidePublicSquadEligibilityCard);
   }, [completeAction]);
 
   const onSubmit = useCallback(() => {
-    openModal({ type: LazyModal.SubmitSquadForReview, props: { squadId } });
-  }, [openModal, squadId]);
+    if (!squad.id) {
+      return;
+    }
 
-  if (hideCard) {
-    return null;
-  }
+    openModal({
+      type: LazyModal.SubmitSquadForReview,
+      props: { squadId: squad.id },
+    });
+  }, [openModal, squad]);
 
-  const isEligible = postsCount >= MIN_POSTS;
-  const effectivePostsCount = isEligible ? MIN_POSTS : postsCount;
+  if (!isFetched) {
+    return <PlaceholderCard />;
+  }
 
+  const isEligible = postsCount >= MIN_SQUAD_POSTS;
   const CardComponent = shouldUseListFeedLayout ? CardV1 : Card;
 
   return (
     <CardComponent
       data-testid="publicSquadEligibilityCard"
-      className="justify-between !bg-transparent p-4"
+      className="min-h-[21.75rem] justify-between !bg-transparent p-4"
```

Comment:
Is this a different min-h from the main card?
won't it force others differently?

Reply:
The min-h for the card is normally much bigger. Should we probably just use the same?

Reply:
well in idea it should be just like the other cards on the feed no need to be different